### PR TITLE
feat: added cookies (without Date back in Daterange)

### PR DESCRIPTION
**Pull request by LPkkjHD**

### CODE REVIEW

**Review by LPkkjHD — COMMENTED** [review]
LGTM

File: website/src/Scoreboard.tsx [hunk 1]
```diff
@@ -314,7 +381,8 @@ const Scoreboard = () => {
                                     (
                                         (inputValue === "" || row.playername.toLowerCase().includes(inputValue.toLowerCase())) &&
                                         (row.score >= minScore) &&
-                                        (new Date(row.timestamp) >= scoreStartDate && new Date (row.timestamp) <= scoreEndDate)
+                                        (new Date(row.timestamp) >= 
+                                        scoreStartDate && new Date(row.timestamp) <= scoreEndDate)
```
Comment (JAWolfs) [inline website/src/Scoreboard.tsx:385]: one line please

**Review by JAWolfs — COMMENTED** [review]
.

**Review by JAWolfs — APPROVED** [review]
lgtm

**Pull request merged**